README.md

```diff
@@ -39,9 +39,9 @@ contains all of the javascript that runs the viewer functionality.
 In order to make things as easy as possible Workiva is now hosting the javascript 
 via a CDN. It can be accessed via the followng CDN url: 
 ```
-https://cdn-prod.wdesk.org/ixbrl-viewer/<tagged version>/ixbrlviewer.js
+https://cdn-prod.wdesk.com/ixbrl-viewer/<version tag>/ixbrlviewer.js
```

Comment:
Smallest of nits: Generally you would do `<version_tag>` to make it clear that its not actually multiple values. This is completely fine though